ksql-rest-app/pom.xml

```diff
@@ -51,6 +52,38 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.curator</groupId>
```

Comment:
Why do we need this dependency on curator? As mentioned before, a description of the problem and the fix would help in understanding this decision.

Reply:
@apurvam added more info on the problem in the description of the PR.

Reply:
I still don't get why we need curator? I don't believe that `EmbeddedSingleNodeKafkaCluster` uses it. It would need a zookeeper dependency

Reply:
EmbeddedSingleNodeKafkaCluster uses ZooKeeperEmbedded which depends on curator.